instrumentation/vibur-dbcp-11.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/viburdbcp/ViburDbcpInstrumentationModule.java

```diff
@@ -5,19 +5,30 @@
 
 package io.opentelemetry.javaagent.instrumentation.viburdbcp;
 
+import static io.opentelemetry.javaagent.extension.matcher.AgentElementMatchers.hasClassesNamed;
 import static java.util.Collections.singletonList;
+import static net.bytebuddy.matcher.ElementMatchers.not;
 
 import com.google.auto.service.AutoService;
 import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
 import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
 import java.util.List;
+import net.bytebuddy.matcher.ElementMatcher;
 
 @AutoService(InstrumentationModule.class)
 public class ViburDbcpInstrumentationModule extends InstrumentationModule {
   public ViburDbcpInstrumentationModule() {
     super("vibur-dbcp", "vibur-dbcp-11.0");
   }
 
+  @Override
+  public ElementMatcher.Junction<ClassLoader> classLoaderMatcher() {
+    // ViburDBCPConfig was renamed in 10.0; this matcher excludes all versions < 10.0
+    // in 11.0, the ViburDBCPDataSource#getPool() method signature was changed - this is detected by
+    // muzzle
+    return not(hasClassesNamed("org.vibur.dbcp.ViburDBCPConfig"));
```

Comment:
Turns out that references collected by muzzle have changed slightly, and no longer contain any mention of `ViburConfig` -- that is because we're no longer passing a method reference `ViburConfig::getPoolMaxSize`, we're calling it inside a lambda function; muzzle collects it as `ViburDBCPDataSource::getPoolMaxSize` (see invokedynamic visitor in `ReferenceCollector`)